docs/runbooks/VirtOperatorDown.md

```diff
@@ -1,27 +1,64 @@
+<!-- Edited by Jiří Herrmann, 9 Nov 2022 -->
+
 # VirtOperatorDown 
 
 ## Meaning
 
-This alert is triggered when there was no virt-operator pod detected in the `Running` state in the past 10 minutes. The virt-operator deployment has a default replica of 2 pods.
+This alert fires when no `virt-operator` pod in the `Running` state has been detected for 10 minutes. 
+
+The `virt-operator` is the first operator to start in a cluster. Its primary responsibilities include the following: 
```

Comment:
Really small nit, but I know that OCP peer review would find this. 
"first operator" > "first Operator"